src/main/java/com/org/jenkins/custom/jenkins/distribution/service/util/Util.java

```diff
@@ -42,4 +47,21 @@ public void cleanupTempDirectory(File file) {
     public JSONObject convertPayloadToJSON(String payload) throws Exception{
         return new JSONObject(payload);
     }
+
+    public ResponseEntity<Resource> returnResource(HttpHeaders headers, File file, InputStreamResource resource){
+        return ResponseEntity.ok()
+            .headers(headers)
+            .contentLength(file.length())
+            .contentType(MediaType.parseMediaType( "application/octet-stream"))
+            .body(resource);
+    }
+
+    public HttpHeaders returnHeaders(String headerValue) {
```

Comment:
Same here, this could be a static method.

Reply:
:+1: